code.gs

```diff
@@ -90,6 +90,7 @@ var settings = {
      * By default the script only tracks birthday events.
      */
     eventTypes: {
+      Birtyhdays: true,
```

Comment:
i believe there is typo

Reply:
typo indeed, my bad i haven't checked will fix asap